### ISSUE TITLE
Admin styling using block ID

### ISSUE THREAD

**Issue by lewisself64** [WordPress/gutenberg#9950]
I’m developing a banner block. I’ve created some options to align the text on the banner to the right or the left. The block creates a <style> tag and adds the styling to the page. However, it effects all blocks when I want it to only target the specific block. I know there is an ID on the block (block-e6c047a1-a36f-49fc-9c15-c6075f0917f5) but I’m not sure how I can get the block ID inside of the JavaScript so I can target each block individually.

Is it possible to access the block id inside of the block it's self or not? If not, would it be a good idea to implement this to help with admin styling?

**Comment by lewisself64** [comment]
I've created a RichText input with a tagName of 'p' and a className of 'content-banner-tagline'. The className gets applied to the <p> tag on the admin. However, the styling I need to make is for the 'editor-rich-text' class. 

As I've been writing this I realised I could probably just make a 'right-alignment' and 'left-alignment' class and apply them on the RichText wrapper depending on the users choice for alignment which would help me figure out which banners need to be left and right aligned?
